modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/PageRenderer/index.js

```diff
@@ -184,6 +184,8 @@ const Renderer = ({
 		PAGE_HEADER_COMPONENT_TYPE[page.headerRenderer] ||
 		Components.PageHeader;
 
+	const hasRequiredFields =  page.rows.filter(x => x && x.columns[0] && x.columns[0].fields[0] && x.columns[0].fields[0].required).length > 0;
```

Comment:
@georgel-pop-lr, does this commit intend to add a warning message for each page which has at least a required field, am I right? 

If so, we intentionally removed it a while ago. Do you want to put it back to make it clear to the Forms users or because you're trying to address some Accessibility issue? Can you please give me more details?

Thanks.

Reply:
Hi @natocesarrego,

Yes correctly, I didn't know there was any implementation before and I did this myself, but the client basically asked us to provide something like that, not sure if we should provide it under a configuration or as a component maybe, could be also and option, but the idea is to provide the user with a visible legend of this icon meaning.

Reply:
Thanks for your clarification, @georgel-pop-lr. 🙂 By the middle of this year, Forms team discussed about it on [LPS-87515](https://issues.liferay.com/browse/LPS-87515) and we decided to not add this feature back, but I just restarted the discussion about it with Forms Team Designer and let's see how it goes. Once I get an answer from him, I let you know.

Reply:
Hi @georgel-pop-lr, after discuss with Forms team Designer about this matter, we agreed that the message should be added back only when a form is rendered to be filled out and not on Form Builder, it's important emphasize it because, on 7.1, the message was displayed on both places. Besides, the message chosen by the team is **Fields with * are required** instead of the proposed by you.

Thanks! 🙂

Reply:
Hi @natocesarrego,

Okay great, but how do I know the difference, is there any attribute I could check?